src/theory/arith/nl/ext_theory_callback.cpp

```diff
@@ -129,7 +129,7 @@ bool NlExtTheoryCallback::isExtfReduced(
       }
     }
   }
-  return true;
+  return false;
```

Comment:
Which cases do change here? It is only PI?

Reply:
Potentially many, the method was blatantly wrong before.  However, we don't really use this value much, it determines what is put into this temporary vector https://github.com/cvc5/cvc5/blob/master/src/theory/arith/nl/nonlinear_extension.cpp#L250.

Reply:
So it does not only change pi?

Reply:
Right, the fix also ensures that other transcendental functions are added to `nred` on that line.